frontend/src/pages/CourseSelector/courseDescription/CourseDescription.jsx

```diff
@@ -85,115 +83,126 @@ export default function CourseDescription() {
   };
 
   return (
-    <div className="cs-description-root">
-      {!pageLoaded ? (
-        <Loading />
-      ) : (
-        <>
-          <div className="cs-description-content">
-            {show && courseInPlanner && (
-              <Alert
-                message={`Successfully added ${id} to your planner!`}
-                type="success"
-                style={{ marginBottom: "1rem" }}
-                showIcon
-                closable
-                afterClose={() => setShow(false)}
-              />
-            )}
-            {show && !courseInPlanner && (
-              <Alert
-                message={`Successfully removed ${id} from your planner!`}
-                type="success"
-                style={{ marginBottom: "1rem" }}
-                showIcon
-                closable
-                afterClose={() => setShow(false)}
-              />
-            )}
-            <div className="cs-desc-title-bar">
-              <Title level={2} className="text">
-                {id} - {course.title}
+    <div>
+      <div className="cs-description-root">
+        {!pageLoaded ? (
+          <Loading />
+        ) : (
+          <>
+            <div className="cs-description-content">
+              {show && courseInPlanner && (
+                <Alert
+                  message={`Successfully added ${id} to your planner!`}
+                  type="success"
+                  style={{ marginBottom: "1rem" }}
+                  showIcon
+                  closable
+                  afterClose={() => setShow(false)}
+                />
+              )}
+              {show && !courseInPlanner && (
+                <Alert
+                  message={`Successfully removed ${id} from your planner!`}
+                  type="success"
+                  style={{ marginBottom: "1rem" }}
```

Comment:
can this styling be pushed to a less file?

Reply:
We encountered an issue where certain ant design components overrode the less styles that we gave. I've now solved this by applying an !important to less style. It's not ideal to use this but I think in this case, we can get away with it.

Reply:
Yea front end libraries are usually a disaster with this type of thing lol - lgtm